app/tests/spec/lib/user-agent.js

```diff
@@ -458,6 +458,25 @@ describe('lib/user-agent', () => {
     });
   });
 
+  describe('genericDeviceType', () => {
+    const typeToGenericType = {
+      desktop: 'desktop',
+      embedded: 'mobile',
+      mobile: 'mobile',
+      smarttv: 'mobile',
```

Comment:
I would think `smarttv` would be closer to a tablet icon than a mobile icon?

Reply:
I put it as mobile because their keyboards are so terrible. I figured it's something we can iterate on, or maybe get @ryanfeeley to provide us with a nice icon since people sign in on Fire TVs.